docs/docs/integrations/llms/azure_ml.ipynb

```diff
@@ -181,7 +181,7 @@
     "content_formatter = CustomFormatter()\n",
     "\n",
     "llm = AzureMLOnlineEndpoint(\n",
-    "    endpoint_api_type=\"realtime\",\n",
+    "    endpoint_api_type=\"dedicated\",\n",
     "    endpoint_api_key=os.getenv(\"BART_ENDPOINT_API_KEY\"),\n",
     "    endpoint_url=os.getenv(\"BART_ENDPOINT_URL\"),\n",
     "    model_kwargs={\"temperature\": 0.8, \"max_new_tokens\": 400},\n",
```

Comment:
Wondering if we should make this strongly typed so we help the developer, but cast to dictionary in case the user wants a custom parameter. Something like `OpenAIInferenceParams(BaseInferenceParams)` and `BaseInferenceParams(dict)`.

Reply:
For now lets keep the same enum. As per our earlier discussion, the goal was to eventually get rid of it anyway because we intend to close the gap on the client side between dedicated and serverless.